src/components/ChartCard/ChartCardHeader-Ring.vue

```diff
@@ -41,7 +41,7 @@ export default {
     mainClasses() {
       return {
         'display-3': this.$vuetify.breakpoint.lgAndUp,
```

Comment:
vuetify's still ok with this one?

Reply:
Hm, probably doesn't know it exists - I'll update it

Reply:
Updated 👍